packages/data-point/lib/reducer-path/factory.js

```diff
@@ -13,27 +13,78 @@ const REDUCER_PATH = (module.exports.type = 'ReducerPath')
 function ReducerPath () {
   this.type = REDUCER_PATH
   this.name = ''
+  this.body = undefined
   this.asCollection = false
 }
 
 module.exports.ReducerPath = ReducerPath
 
+/**
+ * @param {*} source
+ * @returns {boolean}
+ */
 function isPath (source) {
   return _.isString(source) && source.charAt(0) === '$'
 }
 
 module.exports.isPath = isPath
 
+const getAccValue = acc => acc.value
+
+module.exports.getAccValue = getAccValue
+
+const getFromAcc = jsonPath => acc => _.get(acc, jsonPath)
+
+module.exports.getFromAcc = getFromAcc
+
+const getFromAccValue = jsonPath => acc => _.get(acc.value, jsonPath)
+
+module.exports.getFromAccValue = getFromAccValue
+
+const mapFromAccValue = jsonPath => acc => {
+  if (Array.isArray(acc.value)) {
+    return _.map(acc.value, jsonPath)
+  }
+
+  return null
+}
+
+module.exports.mapFromAccValue = mapFromAccValue
```

Comment:
These functions aren't used outside of this file, so perhaps they shouldn't be exported. There's no harm in exporting them, but it does make it a little clearer when a function is not exported that it's not used anywhere else.